e2e-cypress/Jenkinsfile.template

```diff
@@ -44,53 +45,105 @@ odsComponentPipeline(
+
+    // Example for loading environment variables for Azure SSO; please adapt variable names to your OpenShift config,
+    // making sure to precede the variable names with the environment name in lowercase (e.g., dev_username, dev_password,
+    // test_username, test_password, etc.)
+    // cypressUser = sh(returnStdout: true, script:"oc get secret e2euser -o jsonpath='{.data.${context.environment}_username}' | base64 -d")
+    // cypressPassword = sh(returnStdout: true, script:"oc get secret e2euser -o jsonpath='{.data.${context.environment}_password}' | base64 -d")
     // authenticatorOTPSecret = sh(returnStdout: true, script:"oc get secret azure -o jsonpath='{.data.OTP_SECRET}' | base64 -d")
 
     withEnv(["TAGVERSION=${context.tagversion}",
       "NEXUS_HOST=${context.nexusHost}",
       "OPENSHIFT_PROJECT=${context.targetProject}",
       "OPENSHIFT_APP_DOMAIN=${context.getOpenshiftApplicationDomain()}",
-      // "CYPRESS_TENANT_ID=${azureTenantId}",
-      // "CYPRESS_CLIENT_ID=${azureClientId}",
-      // "CYPRESS_CLIENT_SECRET=${azureClientSecret}",
-      // "CYPRESS_USERNAME=${cypressUser}",
-      // "CYPRESS_PASSWORD=${cypressPassword}",
-      // "OTP_SECRET=${authenticatorOTPSecret}",
       "COMMIT_INFO_SHA=${context.gitCommit}",
       "BUILD_NUMBER=${context.buildNumber}",
+      "BASE_URL=${baseUrl}",
     ]) {
-      sh 'npm install'
-      def status = sh(script: 'npm run e2e', returnStatus: true)
-      sh 'npm run combine:reports'
-      junit(testResults:'build/test-results/*.xml', allowEmptyResults: true)
-      stash(name: "installation-test-reports-junit-xml-${context.componentId}-${context.buildNumber}", includes: 'build/test-results/installation-junit.xml', allowEmpty: true)
-      stash(name: "integration-test-reports-junit-xml-${context.componentId}-${context.buildNumber}", includes: 'build/test-results/integration-junit.xml', allowEmpty: true)
-      stash(name: "acceptance-test-reports-junit-xml-${context.componentId}-${context.buildNumber}", includes: 'build/test-results/acceptance-junit.xml', allowEmpty: true)
+      withCredentials([
+        // string(credentialsId: cypressUser, variable: 'CYPRESS_USERNAME'),
+        // string(credentialsId: cypressPassword, variable: 'CYPRESS_PASSWORD'),
+        // string(credentialsId: authenticatorOTPSecret, variable: 'OTP_SECRET')
+      ]) {
+        sh 'npm install'
+
+        // Note: Testing in the production environment is not recommended as it can lead to unintended consequences,
+        // including potential downtime, data corruption, or exposure of sensitive information.
+        // This block is designed to skip acceptance and integration tests in the production environment to avoid these risks.
+        // If you choose to enable these tests in production, do so at your own risk and take all necessary precautions.
+        def status
+        if (context.environment == 'prod') {
```

Comment:
This split between `npm run e2e` and `npm run e2e:prod` might require an update in the documentation here: https://github.com/opendevstack/ods-quickstarters/blob/master/docs/modules/quickstarters/pages/e2e-cypress.adoc?plain=1#L124-L137

Reply:
you are right! Forgot about this part of the documentation, thanks!